### PR TITLE
issue 550 add monitoring to iplan

### PR DESCRIPTION
**Pull request by yaelal**

### CODE REVIEW

File: server/config/development.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "uptimeRobot": {
+		"iplanHeartbeatUrl": "https://www.google.com"
```
Comment (muamichali) [inline server/config/development.json:3]: you could also make it "" in dev and add code that checks if anything is there, and does not try to hit any URLs if the string is empty <3

Comment (yaelal) [reply server/config/development.json:3]: i didnt want code that isnt called, seems better to have it tested for real

File: server/bin/iplan [hunk 2]
```diff
@@ -2,9 +2,22 @@
...
+    const iplanHeartbeatUrl = Config.get('uptimeRobot.iplanHeartbeatUrl');
+    try {
+      const response = await axios.get(iplanHeartbeatUrl);
+      Log.debug("iplan monitor status result: " + response.status);
+      Log.debug("iplan monitor msg result: " + response.data.msg);
+    } catch (error) {
+      Log.error("iplan monitor error msg: " + error.response.body);
+    }
+  })
   .catch(e => Log.error(e))
```
[the remark this answers is not in the document]
Comment (yaelal) [reply server/bin/iplan:22]: no, only one url

**Comment by GalGend** [comment]
Hi @yaelal , please pull from master and push, fixed the tests that failed

**Pull request merged**